app/views/spree/user_mailer/signup_confirmation.en-GB.text.erb

```diff
@@ -4,11 +4,8 @@ Welcome to Open Food Network UK! Your login email is <%= @user.email %>
 
 You can go online and start shopping through food hubs and local producers you like at http://openfoodnetwork.org.uk
 
-We welcome all your questions and feedback; you can use the Send Feedback button on the site or email us at hello@openfoodnetwork.org.
```

Comment:
@hughbric Sorry for the confusion! You did as described in the issue description, but a later [comment](https://github.com/openfoodfoundation/openfoodnetwork/issues/1279#issuecomment-429145869) says to change the text instead:

> Update text to read: "We welcome all your questions and feedback; you can email us at hello@openfoodnetwork.org.au" <--- note that this email will be different for each instance.

Could you update?

For the email address specific to the instance, you can use `ContentConfig.footer_email`, which is set in: **Admin > Configuration > Content > Footer and External Links > Email**.

Reply:
I wonder what this en-GB specific file is, isnt this supposed to be in en-GB in some way?